server/schema/article/article.datasources.js

```diff
@@ -108,6 +108,11 @@ const findByYearAndMonth = (allowRestricted, onlyPublished, limit, offset, start
     },
   ]);
 
+const countNumberOfArticles = (allowRestricted, onlyPublished) =>
+  ArticleModel.countDocuments({
+    $and: [...getBaseConditions(allowRestricted, onlyPublished)],
```

Comment:
Why won't this work? Do we need to copy array?
```suggestion
    $and: getBaseConditions(allowRestricted, onlyPublished),
```